Cargo.toml

```diff
@@ -22,13 +23,17 @@ travis-ci = { repository = "clap-rs/clap_derive" }
 appveyor = { repository = "https://github.com/clap-rs/clap_derive", service = "github" }
 
 [dependencies]
-syn = "0.15.39"
-quote = "0.6"
-proc-macro2 = "0.4"
+syn = { version = "1", features = ["full"] }
+quote = "1"
+proc-macro2 = "1"
+heck = "0.3.0"
 clippy = {version = "0.0.174", optional = true }
```

Comment:
What is this `clippy` dependency for? I suppose a long time ago, in a galaxy far, far away, it had been being used in place of nowadays `cargo clippy`. If so, we can remove it. cc @kbknapp @Dylan-DPC

Reply:
It's for later when we have CI working.

Reply:
ja i don't think we need that anymore.